commodore/lib/commodore.libjsonnet

```diff
@@ -6,6 +6,13 @@ local namespaced(ns, obj) =
     metadata+: { namespace: ns },
   };
 
+/**
+ * \brief A function to filter NULL values.
+ *
+ * \arg `list` The array to filter
+ *
+ * \deprecated Use `std.prune(a)` instead.
+ */
 local filterNull(list) = std.filter(function(obj) obj != null, list);
```

Comment:
I would also produce the deprecation warning when the function is called, as shown below.

```suggestion
local filterNull(list) = std.trace('filterNull is deprecated, use std.prune(a) instead', std.filter(function(obj) obj != null, list));
```